.github/workflows/cd.yaml

```diff
@@ -0,0 +1,24 @@
+name: CD
+
+on:
+  push:
+    tags: ["v*"]
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout Code Repository
+        uses: actions/checkout@v2
+      - name: Set up Python 3.9
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install `pypa/build`
+        run: python -m pip install build
+      - name: Build sdist and wheel
+        run: python -m build --sdist --wheel --outdir ./dist/
+      - name: Publish package to PyPI
+        uses: pypa/gh-action-pypi-publish@master
```

Comment:
Let's use the latest stable release instead of the tip of `master`, as recommended in the [docs](https://github.com/pypa/gh-action-pypi-publish/tree/master#usage):

`pypa/gh-action-pypi-publish@release/v1`